app/models/user_mode/auth0_auth_strategy.rb

```diff
@@ -0,0 +1,22 @@
+class UserMode::Auth0AuthStrategy < UserMode::MultiUser
+  def set_auth_provider(omniauth)
+    omniauth.provider :auth0,
+                      Rails.configuration.auth0_client_id,
+                      Rails.configuration.auth0_client_secret,
```

Comment:
Why not using composition over inheritance?

The abstraction _`auth0` strategy is a `MultiUserMode`_ isn't really sound.